simde/arm/neon/recps.h

```diff
@@ -39,17 +42,7 @@ simde_vrecps_f32(simde_float32x2_t a, simde_float32x2_t b) {
   #if defined(SIMDE_ARM_NEON_A32V7_NATIVE)
     return vrecps_f32(a, b);
   #else
-    simde_float32x2_private
-      r_,
-      a_ = simde_float32x2_to_private(a),
-      b_ = simde_float32x2_to_private(b);
-
-    SIMDE_VECTORIZE
-    for (size_t i = 0 ; i < (sizeof(r_.values) / sizeof(r_.values[0])) ; i++) {
-      r_.values[i] = SIMDE_FLOAT32_C(2.0) - (a_.values[i] * b_.values[i]);
-    }
-
-    return simde_float32x2_from_private(r_);
+    return simde_vsub_f32(simde_vdup_n_f32(SIMDE_FLOAT32_C(2.0)), simde_vmul_f32(a, b));
```

Comment:
How about `simde_vmls_f32(simde_vdup_n_f32(SIMDE_FLOAT32_C(2.0)), a, b)` instead?  That way we have a chance to use `_mm_fnmadd_ps` on x86, and it should still degrade to the same series of operations as you have above on other targets.

Reply:
Good idea, thanks!